.changeset/curly-pillows-decide.md

```diff
@@ -0,0 +1,20 @@
+---
+"@comet/admin": minor
+---
+
+Add new `DataGridToolbar` component
```

Comment:
Should we add some context here why this is needed for data grids?

Reply:
Like this? bd70bae1900ea666edf6606ff05b639c2e2df37b